components/NotFound.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import Header from "./Header";
+
+function NotFoundPage(): React.ReactElement {
+  return (
+    <div className="NotFoundPage">
+      <Header />
+      <header>
+        <h1 className="font-extrabold text-5xl leading-10 tracking-tight text-gray-900">
+          404
+        </h1>
+        <h2 className="mt-4 sm:mt-5 font-light text-2xl text-center leading-tight text-gray-900">
+          <strong className="font-semibold">Deno</strong> and{" "}
+          <strong className="font-semibold">Ferris</strong> couldn't find what
+          you're looking for.
+        </h2>
```

Comment:
Can you replace this h2 text. Either with "not found" or removed entirely.  The "ferris" bit will confuse people not steeped in programming language lore.